### PR TITLE
Feature: Add About Page

### LINKED ISSUE #189: [FEATURE] Add about page, transfer team section and include to overview
### Detailed description

# About

## What is the mission and vision of Dev Protocol

The mission of Dev Protocol is to maximize creativity for every creator in the world, using the power of Blockchain and Decentralized Financial mechanism to enable creators to capitalize on their projects and achieve financial success, regardless of the size of the project.

## Where does the Dev Protocol idea come from?

The idea does not come from Open Source Software (OSS) but traditional work. One of us, the founders, found out that Japanese artisans of temples and shrines were economically less valued despite their elaborate and long-lasting works. Then we got together and started to tackle continuous finance for creators in 2015. We established Dev Protocol in 2018 to begin implementing our sustainable supporting scheme by blockchain technology.

## The value of the Dev Protocol

The positive cycle created by Public goods such as OSS begins sustainability and enables more OSS developers, artists, researchers, and other creators to live independently. To collaborate with more projects freely is the greatest value for all. 

The core features of Dev Protocol are evaluation, distribution and management for the creator's projects. By using Dev Protocol, even a small, newly started project can increase its evaluation and improve its market capitalization. The ability to achieve this has been limited to stock companies now, but Dev Protocol is opening up that possibility to all creators.

The first step for creators is to prove their ownership of the project and issue their own property tokens(ERC-20). Once issue tokens, creators can monetize through staking by supporters, share and divide them with their co-developers. In addition, these tokens allow creators to give partial ownership for all the contributors involved in the OSS development in order to provide a sustainable opportunity for profit without the need of token sales. 

This is just a description of the basic components of the Dev Protocol ecosystem. There are many possible use cases for this new economic infrastructure designed for creators, so please try it out for yourself!

## The Team
image

### Context

This is also important part of our website, best to include in overview nav-item

### Possible implementation

Add about page, the information above ☝️ and remove the question & answer in [FAQ](https://devprotocol.xyz/faq)
image

Remove the team section in [developers](https://devprotocol.xyz/developers) page then transfer it to about page

### Additional information

Thank you :)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by AbhinavMV**
#### Description of the change

Added about page
- Added in navbar (overview)
- Removed Teams Section from developer page
- Removed Q&A from FAQ

Closes #189 

#### Screenshots
image

#### Checklist

**I agree to the following :-**

* [x]   Added description of the change
* [x]   I've read the [contributing guidelines](https://github.com/web3community/devprotocol.xyz/blob/main/CONTRIBUTING.md)
* [x]   Search previous suggestions before making a new PR, as yours may be a duplicate.
* [x]   I acknowledge that all my contributions will be made under the project's license.

Notes (**optional**): 
Let me know if any changes are needed or if you have a better design in mind. 😄

<a href="https://gitpod.io/#https://github.com/web3community/devprotocol.xyz/pull/192"></a>

### CODE REVIEW

File: src/components/About/About.astro [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        The core features of Dev Protocol are evaluation, distribution and management for the creator's projects. By using Dev Protocol, even a small, newly started project can increase its evaluation and improve its market capitalization. The ability to achieve this has been limited to stock companies now, but Dev Protocol is opening up that possibility to all creators.<br/><br/>
+        The first step for creators is to prove their ownership of the project and issue their own property tokens(ERC-20). Once issue tokens, creators can monetize through staking by supporters, share and divide them with their co-developers. In addition, these tokens allow creators to give partial ownership for all the contributors involved in the OSS development in order to provide a sustainable opportunity for profit without the need of token sales.<br/><br/>
+        This is just a description of the basic components of the Dev Protocol ecosystem. There are many possible use cases for this new economic infrastructure designed for creators, so please try it out for yourself!</p>
+    </div>
+  </div>
+</section>
```
Comment (Panquesito7) [inline src/components/About/About.astro:20]: Missing an endline.
```suggestion
</section>

```

File: src/components/About/About.astro [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        The core features of Dev Protocol are evaluation, distribution and management for the creator's projects. By using Dev Protocol, even a small, newly started project can increase its evaluation and improve its market capitalization. The ability to achieve this has been limited to stock companies now, but Dev Protocol is opening up that possibility to all creators.<br/><br/>
+        The first step for creators is to prove their ownership of the project and issue their own property tokens(ERC-20). Once issue tokens, creators can monetize through staking by supporters, share and divide them with their co-developers. In addition, these tokens allow creators to give partial ownership for all the contributors involved in the OSS development in order to provide a sustainable opportunity for profit without the need of token sales.<br/><br/>
+        This is just a description of the basic components of the Dev Protocol ecosystem. There are many possible use cases for this new economic infrastructure designed for creators, so please try it out for yourself!</p>
```
Comment (Panquesito7) [inline src/components/About/About.astro:17]:
```suggestion
        <p>The positive cycle created by Public goods such as OSS begins sustainability and enables more OSS developers, artists, researchers, and other creators to live independently. To collaborate with more projects freely is the greatest value for all.<br/><br/>
        The core features of Dev Protocol are evaluation, distribution, and management for the creator's projects. By using Dev Protocol, even a small, newly started project can increase its evaluation and improve its market capitalization. The ability to achieve this has been limited to stock companies now, but Dev Protocol is opening up that possibility to all creators.<br/><br/>
        The first step for creators is to prove their ownership of the project and issue their property tokens(ERC-20). Once issue tokens, creators can monetize through staking by supporters, share and divide them with their co-developers. In addition, these tokens allow creators to give partial ownership to all the contributors involved in the OSS development to provide a sustainable opportunity for profit without the need for token sales.<br/><br/>
        This is just a description of the basic components of the Dev Protocol ecosystem. There are many possible use cases for this new economic infrastructure designed for creators, so please try it out for yourself!</p>
```

File: src/pages/about.astro [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+import Layout from '@layouts/Default.astro'
+
+import About from '@components/About/About.astro'
+import Team from '@components/About/Team.astro'
+---
+
+<Layout metaTitle="Developers | Dev Protocol">
```
Comment (Panquesito7) [inline src/pages/about.astro:8]:
```suggestion
<Layout metaTitle="About us | Dev Protocol">
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: src/pages/about.astro (hunk 3 above)
Comment (AbhinavMV) [reply src/pages/about.astro]: Done

File: src/components/About/About.astro (hunk 2 above)
Comment (AbhinavMV) [reply src/components/About/About.astro]: Done

File: src/components/About/About.astro (hunk 1 above)
Comment (AbhinavMV) [reply src/components/About/About.astro]: Done

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Review by vinzvinci — APPROVED** [review]
LGTM 👍

**Review by gptkrsh — APPROVED** [review]

**Pull request merged**